### PR TITLE
Configuring and unconfiguring a dhcp server

### LINKED ISSUE #4743: [Networking]Configuring and Unconfiguring a DHCP Server Using Interactive 'dhcpconfig'
#### Proposed title of article
Configuring and Unconfiguring a DHCP Server Using Interactive 'dhcpconfig'

#### Proposed article introduction
When configuring and managing a Dynamic Host Configuration Protocol (DHCP) server, the exact actions to take depend on the network operating system. 
There are several procedures that shows how to work with DHCP in windows and Linux server as I will discuss in this article

#### Key takeaways 
1. How to Configure a DHCP Server (Interactive dhcpconfig)
2. How to Configure a BOOTP Relay Agent (Interactive dhcpconfig)
3. Configuring Networks Using Interactive dhcpconfig
4. How to Configure the Local Network (Interactive dhcpconfig

#### Article quality
This article will discuss step by step process in configuring and unconfiguring DHCP in both Linux and Windows servers as opposed to other articles that covers  windows only.

#### References
N/A

### Templates to use as guides
- [How To Guide Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/how-to-guide-template.md)
- [Software Review Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/software-review-template.md)
- [Tutorial Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/tutorial-template.md)

### PR DESCRIPTION
**Pull request by immah-mul**
#### Formatting and structure checks
- [X] Have you used the correct folder and file structure?
- [X] Is your article properly formatted in Markdown? - 
- [X] Have you used our preferred heading structure? - 
- [X] Does your article follow any of the suggested structure formats? 

- [X] Is your article over 750 words? 
- [X] Is the article you are submitting an in-depth and unique article? Does it go beyond what is in the official docs and what is covered in other blog sites. See these articles as [examples](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md).
- [X] Does your article provide enough value and detail about your topic? 
- [X] Can your article be understood by beginners? 

- [X] Have you included a hero image that is **600x400 pixels**, under **300KB in size**, and have the copyright to use it?
      
#### Grammar & plagiarism checks
- [X] Have you spellchecked and grammar checked your article?
- [X] Please place your article through a 3rd party plagiarism checker. We suggest using Quetext, [this tool](https://www.quetext.com/) is free and has a daily limit.
- [X] Have you checked your article for readability?
- [] Have you added sources for quotes and images that aren't yours?

#### Technical checks
- [X] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**
- [X] Have you checked your code runs correctly and you've highlighted all necessary dependencies for installation?
- [X] Are the software programs and packages you're highlighting in your article up to date, using current versions and not deprecated?

#### Topic suggestion this closes

This closes #4743

### CODE REVIEW

**Comment by immah-mul** [comment]
@M0nicah done all the needed revisions

**Comment by M0nicah** [comment]
hi @immah-mul, nice article.
I would like to point out sth I noticed:

- The commands and code are supposed to be clearly named/labeled. If the code is in Javascript ensure that you indicate it after the first backticks.
- Your article just has steps of what is to be done. I would advise you to at least add introduction paragraphs after each heading. These paragraphs explain what is to be done in that particular section. You can also explain why the step is important or why it should be done, then go ahead and explain how it is done. This will help the reader prepare for what they are going to read next and also flow easily throughout the step.

Kindly implement the changes, hit me up on slack in case of any questions.
Ping me when you are done.

**Note**: While writing a tutorial do not totally assume that your audience knows everything.

**Comment by immah-mul** [comment]
@M0nicah 
I've taken time to make the requested changes. Kindly check

File: content/articles/configuring-a-dhcp-server/index.md [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+- On the DHCP server system, evaluate yourself to the position of superuser
+- To bring up the text-based DHCP configuration menu, type the following command:
+- 
+```bash
+# /usr/sbin/dhcpconfig
+```
+- Then, select Configure DHCP service by typing 1 and pressing return.
+- To configure a remote network, follow the prompts as they appear.
+
+```bash
```
Comment (M0nicah) [inline content/articles/configuring-a-dhcp-server/index.md:117]: kindly explain this part. Is this a command? if yes, kindly explain what it does.

**Review by M0nicah — CHANGES_REQUESTED** [review]
@immah-mul , kindly check how you indicate your commands. I noticed you indicated some steps/instructions as commands.
You can remove that and take screenshots of the results and instructions that the reader is supposed to have/see.

ping me in case of a question.

File: content/articles/configuring-a-dhcp-server/index.md [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+Enаble DHСР/BООTР suрроrt оf netwоrks yоu seleсt? ([Y]/N):Y
+Соnfigure BООTР/DHСР оn lосаl LАN netwоrk: 102.21.0.0? ([Y]/N):N
```
Comment (M0nicah) [inline content/articles/configuring-a-dhcp-server/index.md:119]: This does not seem to be a command

**Review by M0nicah — CHANGES_REQUESTED** [review]
@immah-mul , kindly check how you indicate your commands. I noticed you indicated some steps/instructions as commands.
You can remove that and take screenshots of the results and instructions that the reader is supposed to have/see.

ping me in case of a question.

**Comment by immah-mul** [comment]
@M0nicah I've done the needed revisions

**Comment by Kayere** [comment]
Congratulations for publishing your first article @immah-mul. Please go over the changes made by your peer reviewer and I to help improve the quality of your future articles.

cc @M0nicah

**Comment by hectorkambow** [comment]
@immah-mul can you please DM me via slack 👍  

Thank you

**Pull request merged**